src/Microsoft.DotNet.XHarness.CLI/Commands/Android/AndroidTestCommand.cs

```diff
@@ -39,6 +39,8 @@ protected override ExitCode InvokeCommand(ILogger logger)
     {
         var runner = new AdbRunner(logger);
 
+        runner.ClearAdbLog();
```

Comment:
move this after `AndroidInstallCommand` so we can be sure the device is up and connected.